cloudrun-malware-scanner/config.json

```diff
@@ -9,5 +9,5 @@
     "'ClamCvdMirrorBucket' is a GCS bucket used to mirror the clamav database definition files to prevent overloading the Clam servers",
     "and being rate limited/blacklisted. Its contents are maintained by the updateCvdMirror.sh script"
   ],
-  "ClamCvdMirrorBucket": "cvd-mirror-development-344423"
+  "ClamCvdMirrorBucket": ""
```

Comment:
can we just remove ClamCvdMirrorBucket from this file then?

Reply:
Better leave it here since it's a hard code key for other config files to looking for the value. I don't want to hide too much for clamav implementation.

Reply:
Another reason I keep it here:
If we want to do auto scan and move in the future, the config.json is the ideal interface for configuring buckets(since the bucket should be static instead of changing through deploy). Comparing to the room for future extension, I think it's ok to leave here.